src/moneyTracker.js

```diff
@@ -0,0 +1,35 @@
+function Moneytracker() {
+  this._transactions = [];
+};
+
+Moneytracker.prototype.recordEarning = function (money, date) {
+  this._transactions.push(new Transaction(money, date))
+};
+
+Moneytracker.prototype.recordExpenditure = function (money, date) {
+  this._transactions.push(new Transaction(money, date))
+};
+
+Moneytracker.prototype._balance = function () {
```

Comment:
The name of this method is a bit misleading.  Try describing what it does in a sentence.  This technique sometimes leads to a better name, and sometimes leads to the desire to refactor the method because it's hard to name.